2015/proposals/li-yanghao.md

```diff
@@ -0,0 +1,83 @@
+# Add taxonomic name resolution to the EcoData Retriever to facilitate data-intensive approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is a Python based tool for automatically downloading, cleaning up, restructuring ecological data files into standard formats, and then storing them in your choice of database management systems. It does the hard work of data munging so that scientists can focus on doing science.
+
+This project focuses on the problem that the names of species in ecological data are constantly being redefined. By automating reconciliation of different species names as part of the process of accessing the data in the first place, it will become much easier for users to combine diverse datasets for specific scientific research.
+
+## Technical Details
+
+This project would build a system for sending a species name to one or more of the taxonomic name resolution services and determine the best name to appear in the generated databases. So the project mainly contains three components: accessing the resolution services, determining the election strategy and updating the user interfaces.
```

Comment:
"So the project mainly contains three" -> "The project is composed of three main"